source/create.js

```diff
@@ -37,7 +37,8 @@ const create = defaults => {
 		return defaults.handler(normalizeArguments(url, options, defaults), next);
 	};
 
-	for (const method of defaults.methods) {
+	const methods = defaults.methods.map(method => method.toLowerCase());
```

Comment:
Why are you lowercasing them here, but uppercasing them in `normalize-arguments`?

Reply:
> Why are you lowercasing them here,

Imagine calling `got.POST` instead of `got.post` :P

> but uppercasing them in normalize-arguments?

To keep things consistent; because `error.method` is uppercased :)

Reply:
Nvm. I missed the context. It's late here...

Reply:
That's ok, answering questions is my job :P It's a bit late here too :)